pipelines/HSC/RC2.yaml

```diff
@@ -0,0 +1,340 @@
+description: The DRP pipeline specialized for the HSC RC2 dataset.
+instrument: lsst.obs.subaru.HyperSuprimeCam
+imports:
+  - $DRP_PIPE_DIR/ingredients/DRP.yaml
+  - $FARO_DIR/pipelines/metrics_pipeline_jointcal_fgcm.yaml
+tasks:
+  skyCorr:
+    class: lsst.pipe.drivers.skyCorrection.SkyCorrectionTask
+  jointcal:
+    # Running jointcal is something we want to do for all real-world
+    # instruments, so it doesn't belong in an HSC-specific pipeline, let alone
+    # an RC2-specific one.  We just haven't done the pipeline refactoring that
+    # would fix this, and it's not a huge loss because we haven't actually
+    # gotten jointcal running in production on anything other than HSC (it
+    # might just work, but we certainly can't claim it's supported).
```

Comment:
I'd remove some of this statement. Jointcal should absolutely work in production on DECam and CFHT: I've been testing it on those datasets all along (until I had to remove the DECam tests because of lack of gen3 test data). It should also work on LSST, if we had any data (the simulated data isn't useful). I'd end the sentence at "...that would fix this.", without the paranthetical.